setup.cfg

```diff
@@ -136,6 +139,7 @@ scancodeio_pipelines =
     collect_symbols_tree_sitter = scanpipe.pipelines.collect_symbols_tree_sitter:CollectSymbolsTreeSitter
     enrich_with_purldb = scanpipe.pipelines.enrich_with_purldb:EnrichWithPurlDB
     find_vulnerabilities = scanpipe.pipelines.find_vulnerabilities:FindVulnerabilities
+    get_scorecard_info_packages = scanpipe.pipelines.get_scorecard_info_packages:FetchScoreCodeInfo
```

Comment:
`fetch_scorecode_info` would be much better and consistent with the pipeline name.